grammar/FuncTestCaseLexer.g4

```diff
@@ -0,0 +1,103 @@
+lexer grammar FuncTestCaseLexer;
+
+import SubstraitLexer;
+
+options {
+    caseInsensitive = true;
+}
+
+Whitespace    : [ \t\n\r]+ -> channel(HIDDEN) ;
+
+SubstraitScalarTest: '### SUBSTRAIT_SCALAR_TEST:';
+SubstraitInclude: '### SUBSTRAIT_INCLUDE:';
```

Comment:
This feels more like a pragma than a token.  Perhaps change this to a grammar rule like '###' Identifier ':'?

Reply:
Changed it to a parser rule, but retained the `SUBSTRAIT_*` keywords as tokens.